scheduler/src/cook/mesos/mesos_compute_cluster.clj

```diff
@@ -310,6 +310,10 @@
     (async/go
       (async/>! (pool->offers-chan pool-name) offers)))
 
+  (get-outstanding-synthetic-pods [_ _] nil)
```

Comment:
This placeholder is not type compatible with the other implementation, which returns a seq. In the new proposal, this returns 0.

Reply:
I looked up whether it's better to return an empty seq or nil, and I found some suggestion that nil is better. I can update it to return an empty sequence. Would `(sequence ())` be the best approach in that case?

Reply:
`(list)`  but see my second proposal.